### ISSUE TITLE
Inspection records do not contain violation types

### ISSUE THREAD

**Issue by geneorama** [Chicago/sdp-feature-fetcher#3]

**Comment by tomschenkjr** [comment]
@geneorama - have you reached out to the GIVA team on this?

**Comment by geneorama** [comment]
@tomschenkjr No, but I can. Who would be the right person to talk to on their team, probably Viveck?

**Comment by tomschenkjr** [comment]
Yeah

**Comment by geneorama** [comment]
Heard back from UIC this afternoon.  

He didn't want to do a phone call and said that he was busy, but asked about the issues. I said that 

1. The current example only contains two records, and that I could use help running the code to generate a bigger example
2. The business license type is not described
3. The inspection results are missing

In writing the email I did notice that the first example business "BMG HARVEST II, INC." has two license types, and these both appear in the JSON file.  However they do not have the license descriptions or license numbers, so there is not obvious way to distinguish them.  Also, the inspections are not tied to the license information, so there is no way to tell that the food inspection pertains to the “Retail Sales of Perishable Foods” license and not the “Consumption of Liquor on Premises” license.

**Comment by geneorama** [comment]
Added a copy of the example data to `.\doc\examp.json` also added a modified version that contains a fake "result" field in `.\doc\examp_mod.json`

**Comment by geneorama** [comment]
moving example data from `./doc` to `./data`
